x-pack/docs/en/ilm/update-lifecycle-policy.asciidoc

```diff
@@ -1,5 +1,432 @@
+      "skip": false,
+      "lifecycle_date": 1538475653281,
+      "phase": "warm",
+      "phase_time": 1538475653317,
+      "action": "forcemerge",
+      "action_time": 1538475653317,
+      "step": "forcemerge",
+      "step_time": 1538475653317,
+      "phase_execution": {
+        "policy": "my_executing_policy",
+        "modified_date_in_millis": 1538475653317,
+        "version": 2,
+        "phase_definition": {
+          "minimum_age": "1d",
+          "actions": {
+            "forcemerge": {
+              "max_num_segments": 1
+            }
+          }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TESTRESPONSE[s/"lifecycle_date": 1538475653281/"lifecycle_date": $body.indices.my_index.lifecycle_date/]
+// TESTRESPONSE[s/"phase_time": 1538475653317/"phase_time": $body.indices.my_index.phase_time/]
+// TESTRESPONSE[s/"action_time": 1538475653317/"action_time": $body.indices.my_index.action_time/]
+// TESTRESPONSE[s/"step_time": 1538475653317/"step_time": $body.indices.my_index.step_time/]
+// TESTRESPONSE[s/"modified_date_in_millis": 1538475653317/"modified_date_in_millis": $body.indices.my_index.phase_execution.modified_date_in_millis/]
+
+`my_index` will move to the next phase in the latest policy definition, which is the updated `warm` phase.
+
+C. Effects of switching which policy is managing an index
+ - This behaves much like a policy update, but instead of just switching to a different
+     version, it switches to a different policy (the latest version of that policy).
+
+After setting a policy for an index, we can switch out `my_policy` with
+`my_other_policy` by just updating the index's `index.lifecycle.name`
+setting to the new policy. After completing its currently executed phase,
+it will move on to the next phase in `my_other_policy`. So if it was on the
+`hot` phase before, it will move to the `delete` phase after the `hot` phase concluded.
+
+////
+[source,js]
+------------------------
+PUT _ilm/my_policy
+{
+  "policy": {
+    "phases": {
+      "hot": {
+        "actions": {
+          "rollover": {
+            "max_size": "25GB"
+          }
+        }
+      },
+      "delete": {
+        "minimum_age": "10d", <1>
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+
+PUT _ilm/my_other_policy
+{
+  "policy": {
+    "phases": {
+      "delete": {
+        "minimum_age": "1d",
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+
+PUT my_index
+------------------------
+// CONSOLE
+
+////
+
+[source,js]
+--------------------------------------------------
+PUT my_index/_ilm/my_other_policy
+--------------------------------------------------
+// CONSOLE
+// TEST[continued]
+
+////
+[source,js]
+--------------------------------------------------
+{
+  "has_failures": false,
+  "failed_indexes": []
+}
+--------------------------------------------------
+// CONSOLE
+// TESTRESPONSE
```

Comment:
I think it would be good to show the steps like before where the policy doesn't change initially but does when the phase changes

Reply:
maybe there are enough examples of the explain api in the previous sections? I am worried it is too much information at once. I will definitely add more of a note mentioning what to expect here